news/utils.py

```diff
@@ -485,8 +498,11 @@ def get_best_language(languages):
     if not languages:
         return None
 
+    # try again with 2 letter languages
+    languages_2l = [lang[:2] for lang in languages]
```

Comment:
Would you mind confirming that we don't need to worry about 3 letter language codes here?